lib/active_admin/views/components/dropdown_menu.rb

```diff
@@ -44,17 +45,15 @@ def item(*args, **kwargs)
       private
 
       def build_button(name, button_options)
-        button_options[:class] ||= ""
-        button_options[:class] << " dropdown_menu_button"
+        button_options[:class] = "#{button_options[:class]} dropdown_menu_button" || "dropdown_menu_button"
```

Comment:
Is the `|| "dropdown_menu_button"` necessary?

Reply:
nah, we can toss that in the trash